graphql_api/types/plan/plan.py

```diff
@@ -48,6 +49,12 @@ def resolve_plan_name(plan_service: PlanService, info) -> PlanName:
     return plan_service.plan_name
 
 
+@plan_bindable.field("tierName")
+@convert_kwargs_to_snake_case
+def resolve_tier_name(plan_service: PlanService, info) -> TierName:
+    return plan_service.tier_name
```

Comment:
This is where you could call `.value` on the enum.  The return type would be `str`.